bootloader_shell.ld

```diff
@@ -142,10 +142,15 @@ SECTIONS
     __binary_info_end = .;
     . = ALIGN(4);
 
-   .ram_vector_table (NOLOAD): {
+    .ram_vector_table (NOLOAD): {
```

Comment:
nit: you added a space to the start of this line in both linker scripts

Reply:
That's right, I aligned it with the other entries while I was at it.